index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tribute</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <main id="main">
+    <h1 id="title">Tribute to Henry Ford</h1>
+    <figure id="img-div">
+      <img id="image" src="images\Henry_ford_1919.jpg" alt="Henry Ford"/> 
+      <figcaption id="img-caption">Henry Ford</figcaption>
+    </figure>
+    <article id="tribute-info"></article>
+      <p>
+          Henry Ford (July 30, 1863 - April 7, 1947) was an American industrialist and business 
+          magnate. He was the founder of Ford Motor Company, and chief developer of the assembly 
+          line technique of mass production. Ford created the first automobile that middle-class 
+          Americans could afford, and his conversion of the automobile from an expensive luxury into 
+          an accessible conveyance profoundly impacted the landscape of the 20th century.
+      </p>
+      <a href="https://en.wikipedia.org/wiki/Henry_Ford" id="tribute-link" target="_blank">More on Henry Ford </a>
+    </article>
+  </main>
+</body>
+</html>
+</body>
+</html>
```

Comment:
Kindly consider removing the extra **html** and **body** tags at the end of your code. Excessive and unused tags can make your codebase confusing and untidy. Your attention to this suggestion would greatly contribute to maintaining a clean and well-organized codebase.